articles/shared/shared-how-access-centos-update-servers.md

```diff
@@ -19,117 +19,168 @@ toc_mdlink: shared-how-access-centos-update-servers.md
 
 ## Overview
 
-Each of the two security domains within the UKCloud cloud presents its own challenges regarding licensing CentOS machines. This document explains how to access the CentOS repository servers to receive updates.
+Each of the two security domains within the UKCloud cloud presents its own challenges regarding patching and updating CentOS machines. This document explains how to access the CentOS repository servers to receive updates.
 
 Before you attempt to establish a connection to the CentOS repo servers, you need to make sure your virtual machines (VMs) can communicate with the CentOS server, which exists outside of your cloud organisation.
 
 This may involve editing your NAT and firewall settings within your edge gateway to allow traffic to traverse into your virtual data centre (VDC). For how to do this, see the [*How to create NAT rules*](../vmware/vmw-how-create-nat-rules.md) and [*How to create firewall rules*](../vmware/vmw-how-create-firewall-rules.md).
 
 ## Assured OFFICIAL platform
 
-UKCloud's Assured OFFICIAL security domain is internet facing, so you need to configure your VM to connect to the internet, and use a standard update tool, such as Spacewalk RHN, to get updates for CentOS.
+UKCloud's Assured OFFICIAL security domain is internet facing, so you have the option to configure your VM to connect to the internet, and use a standard update tool (such as Spacewalk RHN) or the pubicly accessible CentOS mirrors or you can use UKCloud-managed repositories.
 
 ## Elevated OFFICIAL platform
 
 Our Elevated OFFICIAL security domain doesn't natively connect to the internet, and the PSN network doesn't have any CentOS repo servers. To receive CentOS updates, you can use UKCloud-managed repository servers or a Walled Garden. Both options are described below.
 
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories and EPEL (Extra Packages for Enterprise Linux) for CentOS 6 and 7 on our Elevated OFFICIAL security domain.
+We provide both CentOS standard repositories and EPEL (Extra Packages for Enterprise Linux) for CentOS 6, 7  and 8 on our Elevated OFFICIAL security domain.
 
-To access CentOS base files, create a file called `centos_<x>_il3.repo` in `/etc/yum.repos.d`, where `<x>` is either 6 or 7 depending on your version of CentOS, and populate it with the following:
+>[!NOTE]
+>Please be aware CentOS6.x is deprecated! You can view details on the CentOS Public Mirror Site [here](http://mirror.centos.org/centos/6.10/readme)
+
+For both Assured and Elevated OFFICIAL security domains, you can configure your systems to reference the following server `https://rh-cds.ukcloud.com` In Assured, this domain is configured in DNS so you will automatically be able to resolve it. For Elevated, you will need to UKCloud Support to get the correct IP address and add an entry to `/etc/hosts` on your systems to be able to resolve it.  
```

Comment:
Line 43, missing "Contact" at * in "between you will need to * UKCloud Support"

Reply:
@mikepukcloud - fixed this